site/content/entry/yuzu-progress-report-mar-2020/index.md

```diff
@@ -0,0 +1,131 @@
+With this feature in place, distant textures now look much sharper, improving game quality even over what the original Switch does! 
+Games like Pokémon Sword, Pokémon Shield, and Fire Emblem Three Houses benefit a lot from this.
+However, some other games don’t respond well to this new feature, one example being Astral Chain. Feel free to experiment with it!
+
+An example in Pokémon Sword:
+
+{{< imgs
+    "./15.png| AF Off"
+    "./14.png| AF 16x"
+  >}}
+  
+And another example in Fire Emblem Three Houses:  
+  
+{{< imgs
+    "./01.png| AF Off"
+    "./02.png| AF 8x"
+  >}}
+
+Shader cache invalidations are something our users don’t like at all. 
+We don't too, but it’s all part of the process of improving yuzu to achieve better performance and accuracy.
+For example, one of our recent invalidations was needed to fix the reflections in Luigi’s Mansion 3, making certain parts of the game playable.
+
+Because dancing is important, [namkazt](https://github.com/namkazt) and [Rodrigo](https://github.com/ReinUsesLisp) have been fighting with the code, allowing us to enjoy 3D graphics in Hatsune Miku: Project DIVA MEGA39's, and as collateral, fixed some rendering issues in Bayonetta 1 and Super Smash Bros. Ultimate.
+
+{{< imgs
+    "./07.png| Miku Miku Dance!"
+  >}}
+
+{{< single-title-imgs
+    "Super Smash Bros. Ultimate's menu (Before & After)"
+    "./08.png"
+    "./09.png"
+  >}}
+
+[Kewlan](https://github.com/Kewlan), a regular in our Discord Server, submitted a small change to our input mapping GUI, changing the position of the shoulder buttons so they no longer make anyone angry.
+
+{{< imgs
+    "./11.png| Incorrect."
+    "./12.png| Much better."
+  >}}
+
+We also have a cool improvement by [Tobi](https://github.com/FearlessTobi).
+The Switch is a surround sound compatible console and our implementation of downmixing removed any channel that wasn’t left or right.
+By keeping the extra channels and mixing them in the left and right stereo outputs, the sound in tons of games, including Luigi’s Mansion 3, Sonic Forces, and Bayonetta, has been improved.
+
+## A peculiar case of "Even a broken clock is right twice a day!"
+
+The Nintendo Switch supports `ASTC`, a type of texture format generally used in mobile hardware such as phones and tablets in order to reduce bandwidth usage (a limited resource in those devices).
+Since it's a heavily compressed format, decoding it via the CPU, like it is normally done on PC, is a time-consuming task.
+
+To give an example, Astral Chain is a game that uses `ASTC` very extensively and in bizarre ways, as previous fixes and optimizations have shown.
+On NVIDIA and AMD graphics hardware, the first scene of a new game normally takes typically `17` seconds to load. The GPU has to wait for the CPU to decode all the `ASTC` textures first, thus, slowing down the whole process.
```

Comment:
`normally` and `typically` are redundant. 

You can make the sentence either of the two:

`On NVIDIA and AMD graphics hardware, the first scene of a new game normally takes `17` seconds to load.`
or
`On NVIDIA and AMD graphics hardware, the first scene of a new game typically takes `17` seconds to load.`